### DISCUSSION TITLE
Disabling webpack 5 message

### DISCUSSION THREAD

**Discussion by kimroen** [vercel/next.js#26804] [Help]
We're getting this message when running the next dev server, as expected:

> Using webpack 5. Reason: Enabled by default https://nextjs.org/docs/messages/webpack5

That's great and very appreciated. However, I'm now good with enabling webpack5 and don't need to see this. I tried adding `webpack5: true` to `next.config.js`, but the message is still showing up.

Is there a way to disable the message? If not, I'd suggest that we don't need to show it when webpack 5 is explicitly enabled. The "reason:" part of the message isn't accurate any more when that's the case.

Thoughts/tips?

**Comment by timneutkens** [comment]
It's intentionally mentioned in the 11.0 release to make sure people did not get confused that had already enabled it. We'll remove it at a later point.
